src/components/Avatar.vue

```diff
@@ -0,0 +1,49 @@
+<!-- eslint-disable-next-line vue/no-v-html -->
+<template>
+  <img
+    v-if="!error"
+    :src="avatar"
+    @error="error = true"
+  >
+  <div
+    v-else
+    class="user-identicon"
+    v-html="identicon"
+  />
+</template>
+
+<script>
+import jdenticon from 'jdenticon';
+import defaultAvatar from '../assets/userAvatar.svg';
+import Backend from '../utils/backend';
+import { IDENTICON_CONFIG } from '../utils/util';
+
+export default {
+  props: {
+    address: {
+      type: [String, Object],
+      required: true,
+    },
+  },
+  data: () => ({
+    defaultAvatar,
+    error: false,
+  }),
+  computed: {
+    avatar() {
+      const userImage = Backend.getProfileImageUrl(this.address);
+      return !this.address ? this.defaultAvatar : userImage;
```

Comment:
Shouldn't `this.defaultAvatar` be somehow replaced by identicon()

Reply:
I would just return null, and check that in the template, where it switches to rendering the .svg result